File: 백준/Bronze/9063. 대지/대지.py
N = int(input())

if N == 1:
    X, Y = map(int, input().split())
    print(0)
else:
    minX = 10001
    minY = 10001
    maxX = -10001
    maxY = -10001
    for i in range(N):
        X, Y = map(int, input().split())
        if X < minX:
            minX = X
        if X > maxX:
            maxX = X
        if Y < minY:
            minY = Y
        if Y > maxY:
            maxY = Y
    print((maxX - minX) * (maxY - minY))

File: 백준/Bronze/9063. 대지/대지.java
import java.io.*;
import java.util.*;

public class Main {
	public static void main(String[] args) throws IOException {
		Scanner sc = new Scanner(System.in);
		int N = sc.nextInt();
		if (N == 1) {
			int X = sc.nextInt();
			int Y = sc.nextInt();
			System.out.println(0);
		} else {
			int minX = 10001;
			int minY = 10001;
			int maxX = -10001;
			int maxY = -10001;
			for (int i = 0; i < N; i++) {
				int X = sc.nextInt();
				int Y = sc.nextInt();
				if (X < minX)
					minX = X;
				if (X > maxX)
					maxX = X;
				if (Y < minY)
					minY = Y;
				if (Y > maxY)
					maxY = Y;
			}
			System.out.println((maxX - minX) * (maxY - minY));
		}
		sc.close();
	}
}